js/docgist.js

```diff
@@ -20,18 +16,39 @@ function DocGist($) {
         $gistId.keydown(gist.readSourceId);
     });
 
-    function renderContent(content, link) {
+    function getAsciidoctorOptions() {
+        return Opal.hash2(['to_file', 'safe', 'attributes'], {
```

Comment:
We've learned recently that it is possible to use Opal.hash and then pass in a regular json notation.

```
return Opal.hash({to_file: false, safe: 'secure', attributes: ...})
```

We can't quite figure out why we thought we had to use hash2, other than it was a programming error that got propagated. Both ways work, but clearly Opal.hash is just simpler :)